plans_console.py

```diff
@@ -820,12 +884,20 @@ def doOperClicked(self):
         selFeatureTitle=self.ui.selFeature.text()
         ## check that the shapes exist
         selFeature=self.sts.getFeature(title=selFeatureTitle,featureClassExcludeList=['Folder','OperationalPeriod'])
+        if selFeature == -1:
```

Comment:
why is this needed? sartopo_python already logs a warning if there are multiple matches